crates/velato/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "velato"
+version = "0.0.1"
+description = "Lottie renderer built on vello."
+categories = ["rendering", "graphics"]
+keywords = ["2d", "vector-graphics", "animation", "lottie"]
+
+license.workspace = true
+edition.workspace = true
+repository.workspace = true
+
+[dependencies]
+vello = { path = "../../" }
+bodymovin = { git = "https://github.com/dfrg/bodymovin-rs", rev= "c156dda9" }
```

Comment:
Did you fork `bodymovin-rs` because the upstream repository is unmaintained? Do you plan on taking over the `bodymovin` crate?

Reply:
Ah, missed your explanation in the PR description.